m4/odp_dpdk.m4

```diff
@@ -44,7 +47,6 @@ AS_IF([test "x$DPDK_SHARED" = "xyes"], [dnl
     # rearranged by libtool
     DPDK_LIBS_LT="`echo $DPDK_LIBS | sed -e 's/^/-Wc,/' -e 's/ /,/g'`"
     DPDK_LIBS_LT="$DPDK_LDFLAGS $DPDK_LIB $DPDK_LIBS_LT $DPDK_LIBS"
-    DPDK_LIBS_LT_STATIC="$DPDK_LIBS_LT"
```

Comment:
It seems like `DPDK_LIBS_LT_STATIC` is not used anymore, so it could be removed completely (still couple usages in this file).

Reply:
Done in V8